react/src/components/InputField/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import './styles.css';
+
+const InputField = ({
+  input,
+  label,
+  type,
+  meta: { touched, error, warning }
+}) => (
+  <div className="input-field-group">
+    <h4 className="input-field-title">{label}</h4>
+    <input
+      {...input}
+      placeholder={label}
+      type={type}
+      className={`input-field ${touched && error && 'input-field-error'}`}/>
```

Comment:
Just checking, if `touched` is `false`, will this append `false` as a `className` ?